h/api/search/index.py

```diff
@@ -41,7 +41,7 @@ def index(es, annotation, request):
     # FIXME: this should not use the same presenter as we use to render
     # annotations for clients. It is useful in the mean time until we get rid of
     # the legacy ElasticSearch annotation storage.
-    annotation_dict = presenters.AnnotationJSONPresenter(
+    annotation_dict = presenters.AnnotationSearchIndexPresenter(
         request, annotation).asdict()
 
     annotation_dict['target'][0]['scope'] = [
```

Comment:
If you do, you can move it in as `uri_normalized` rather than scope and make the appropriate changes to the search query builder.